tools/UpdatePaths/Scripts/68241_renames_paicard.txt

```diff
@@ -0,0 +1,3 @@
+# comment finally changed paicard to pai_card
+
+/obj/item/paicard : /obj/item/pai_card
```

Comment:
```suggestion
/obj/item/paicard : /obj/item/pai_card{@OLD}
```

If you don't include `{@OLD}`, all of the nice pixel shifting that's been done so far will delete itself and then all the maps will look ugly.